.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions"
```

Comment:
Shipkit doesn't use github actions

Reply:
> Shipkit doesn't use github actions

is Shipkit going to migrate away from TravisCI?  

The Mockito-Core project appears to be moving to GitHub Actions:
 
https://github.com/mockito/mockito/tree/release/3.x/.github/workflows